compiler/src/dmd/builtin.d

```diff
@@ -167,6 +173,24 @@ Expression eval_unimp(Loc loc, FuncDeclaration fd, Expressions* arguments)
     return null;
 }
 
+Expression eval_ctfeWrite(Loc loc, FuncDeclaration fd, Expressions* arguments)
+{
+    import core.stdc.stdio: fprintf, stderr;
+    import dmd.expression: CTFEExp;
+    import dmd.ctfeexpr: resolveSlice;
+
+    Expression e = (*arguments)[0];
+    if (auto se = resolveSlice(e).toStringExp())
+    {
+        const slice = se.peekString();
+        fprintf(stderr, "%.*s", cast(int)slice.length, slice.ptr);
+    }
+    else
+        fprintf(stderr, "%s", e.toChars());
```

Comment:
hmm, I copied this part, but perhaps indeed this `else` branch can never occur.

Reply:
I assume you can't do `__ctfeWrite(1)`?

Reply:
Indeed you can't. The signature is `void __ctfeWrite(scope const(char)[] s)`

Reply:
I'm checking the necessity of this on Weka's codebase

Reply:
tested and does not seem necessary, simplified the PR.